src/index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
+import './style.css';
+import './assets/images/refresh.png';
+import './assets/images/arrow.png';
+import './assets/images/dot.png';
+import './assets/images/trash-can.png';
+import userActionHandler from './modules/userActions.js';
+import { initStorage } from './modules/localStorage.js';
+import taskStatus from './modules/taskStatus.js';
+import butonHandler from './modules/clearAll.js';
+
+butonHandler;
```

Comment:
Please consider importing this module as a side-effect-only module. Since you are not actually using the butonHanlder you can just:

`import './modules/clearAll.js';`

Reply:
Solved 👍🏽